src/pudl/package_data/eia860/column_maps/emission_control_strategies.csv

```diff
@@ -52,20 +52,20 @@ steam_plant_type_code,,,,,,steam_plant_type,steam_plant_type,steam_plant_type,st
-mercury_emission_wet_scrubber,mercury_emission_ws,mercury_emission_ws,mercury_emission_ws,mercury_emission_wet_scrubber,,,,,,,,,
-mercury_emission_baghouse_shake_and_deflate,,,mercury_emission_bs,mercury_emission_baghouse_shake_and_deflate,,,,,,,,,
-mercury_emission_baghouse_pulse,,,mercury_emission_bp,mercury_emission_baghouse_pulse,,,,,,,,,
-mercury_emission_baghouse_reverse_air,,,mercury_emission_br,mercury_emission_baghouse_reverse_air,,,,,,,,,
-mercury_emission_electrostatic_precipitator_cold_side_with_flue_gas_conditioning,,,mercury_emission_ec,mercury_emission_electrostatic_precipitator_cold_side_with_flue_gas_conditioning,,,,,,,,,
-mercury_emission_electrostatic_precipitator_hot_side_with_flue_gas_conditioning,,,mercury_emission_eh,mercury_emission_electrostatic_precipitator_hot_side_with_flue_gas_conditioning,,,,,,,,,
-mercury_emission_electrostatic_precipitator_hot_side_without_flue_gas_conditioning,,,mercury_emission_ek,mercury_emission_electrostatic_precipitator_hot_side_without_flue_gas_conditioning,,,,,,,,,
-mercury_emission_electrostatic_precipitator_cold_side_without_flue_gas_conditioning,,,mercury_emission_ew,mercury_emission_electrostatic_precipitator_cold_side_without_flue_gas_conditioning,,,,,,,,,
-mercury_emission_other,mercury_emission_ot,mercury_emission_ot,mercury_emission_ot,mercury_emission_other,,,,,,,,,
+ACI,mercury_emission_acj,mercury_emission_acj,mercury_emission_acj,mercury_emission_activated_carbon_injection,,,,,,,,,
+BH,mercury_emission_bh,mercury_emission_bh,mercury_emission_bh,mercury_emission_baghouse,,,,,,,,,
+DS,mercury_emission_ds,mercury_emission_ds,mercury_emission_ds,mercury_emission_dry_scrubber,,,,,,,,,
+EP,mercury_emission_ep,mercury_emission_ep,mercury_emission_ep,mercury_emission_electrostatic_precipitator,,,,,,,,,
+FGD,mercury_emission_fgd,mercury_emission_fgd,mercury_emission_fgd,mercury_emission_flue_gas_desulfurization,,,,,,,,,
+LIJ,mercury_emission_li,mercury_emission_li,mercury_emission_li,mercury_emission_lime_injecton,,,,,,,,,
+WS,mercury_emission_ws,mercury_emission_ws,mercury_emission_ws,mercury_emission_wet_scrubber,,,,,,,,,
+BS,,,mercury_emission_bs,mercury_emission_baghouse_shake_and_deflate,,,,,,,,,
+BP,,,mercury_emission_bp,mercury_emission_baghouse_pulse,,,,,,,,,
+BR,,,mercury_emission_br,mercury_emission_baghouse_reverse_air,,,,,,,,,
+EC,,,mercury_emission_ec,mercury_emission_electrostatic_precipitator_cold_side_with_flue_gas_conditioning,,,,,,,,,
+EH,,,mercury_emission_eh,mercury_emission_electrostatic_precipitator_hot_side_with_flue_gas_conditioning,,,,,,,,,
+EK,,,mercury_emission_ek,mercury_emission_electrostatic_precipitator_hot_side_without_flue_gas_conditioning,,,,,,,,,
+EW,,,mercury_emission_ew,mercury_emission_electrostatic_precipitator_cold_side_without_flue_gas_conditioning,,,,,,,,,
+OT,mercury_emission_ot,mercury_emission_ot,mercury_emission_ot,mercury_emission_other,,,,,,,,,
```

Comment:
Are these codes selected to match the ones that show up in later years? `ACI` and `LIJ` seem to be kind of inconsistent (but if that's the way the latter codes are organized, so be it!)

Reply:
Yes, ACI and LIJ are re-coded to match the dictionary provided in the latest 860 data.